pagefaultgames/pokerogue
# [Sprite] Chien-Pao sprite cleanup

## What are the changes the user will see?
Files the user never sees are now no longer stored in the repository.

## Why am I making these changes?
These files are in the wrong format (both name and sprite dimensions) and do not have an accompanying animation .json, so are undergoing garbage collection.
![image](https://github.com/user-attachments/assets/1cda10b8-cb58-46d5-a600-56e2d3abb6ac)
![image](https://github.com/user-attachments/assets/0e792922-c4b4-4bf6-b698-8826499813f5)
![image](https://github.com/user-attachments/assets/7a8dc266-48a8-43f1-bf9e-c057d0368755)

## What are the changes from a developer perspective?
Developers are no longer able to draw incorrect assumptions about formats based on assets that the game is not currently capable of loading.

## How to test the changes?
Ensure the files no longer exist in the folder `\pokemon\exp\shiny`, I suppose.
Otherwise, if before-testing manages to use the deleted files, I will be utterly flabbergasted.

## Checklist
~~- [ ] Have I considered writing automated tests for the issue?~~
~~- [ ] If I have text, did I make it translatable and add a key in the English locale file(s)?~~
~~- [] Are the changes visual?~~
  ~~- [ ] Have I provided screenshots/videos of the changes?~~
